homeassistant/components/ollama/config_flow.py

```diff
@@ -60,55 +61,51 @@
 _LOGGER = logging.getLogger(__name__)
 
 
-STEP_USER_DATA_SCHEMA = vol.Schema(
-    {
-        vol.Required(CONF_URL): TextSelector(
-            TextSelectorConfig(type=TextSelectorType.URL)
-        ),
-    }
-)
-
-
 class OllamaConfigFlow(ConfigFlow, domain=DOMAIN):
     """Handle a config flow for Ollama."""
 
-    VERSION = 2
-    MINOR_VERSION = 2
+    VERSION = 3
+    MINOR_VERSION = 1
 
     def __init__(self) -> None:
         """Initialize config flow."""
         self.url: str | None = None
-        self.model: str | None = None
-        self.client: ollama.AsyncClient | None = None
-        self.download_task: asyncio.Task | None = None
+
+    @callback
+    def _get_user_schema(self, user_input: dict[str, Any] | None) -> vol.Schema:
+        """Return the schema for the user step."""
+        if user_input is None:
+            user_input = {}
+
+        return vol.Schema(
+            {
+                vol.Required(
+                    CONF_URL,
+                    description={"suggested_value": user_input.get(CONF_URL, "")},
+                ): TextSelector(TextSelectorConfig(type=TextSelectorType.URL)),
+            }
+        )
```

Comment:
Instead of this, add the `STEP_USER_DATA_SCHEMA` back and use the helper `add_suggested_values_to_schema` to add the URL.